ui/src/app/applications/components/application-node-info/container-image-details.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import {useEffect, useState} from 'react';
+import {services} from '../../../shared/services';
+import {GetImageResponse} from '../../../shared/models';
+import {ErrorNotification} from 'argo-ui';
+import Moment from 'react-moment';
+
+export const ContainerImageDetails = ({image}: {image: string}) => {
+    const [error, setError] = useState<Error>();
+    const [resp, setResp] = useState<GetImageResponse>();
+    useEffect(() => {
+        services.images
+            .get(image)
+            .then(setResp)
+            .catch(setError);
+    }, [image]);
+
+    const parts = image.replace(/:.*/, '').split('/');
+    const repo = (parts.length === 1 ? '_/' : '') + parts.join('/');
```

Comment:
Not sure I follow this bit... I feel like we oughta use the `URL` object to manipulate URL-like things.

Reply:
This is a hack create the right URL for hub.docker.com when the image is does no contain slashes, you need to put "_/" into the URL..